src/tests/ftest/datamover/copy_procs.py

```diff
@@ -0,0 +1,130 @@
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+'''
+from data_mover_test_base import DataMoverTestBase
+from os.path import join, sep
+
+class CopyProcsTest(DataMoverTestBase):
+    # pylint: disable=too-many-ancestors
+    """Test class for Datamover multiple processes.
+    Test Class Description:
+        Tests multi-process (rank) copying of the datamover utility.
+        Tests the following cases:
+            Copying with varying numbers of processes (ranks).
+    :avocado: recursive
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Initialize a CopyBasicsTest object."""
+        super(CopyProcsTest, self).__init__(*args, **kwargs)
+
+    def setUp(self):
+        """Set up each test case."""
+        # Start the servers and agents
+        super(CopyProcsTest, self).setUp()
+
+        # Get the parameters
+        self.test_file = self.params.get(
+            "test_file", "/run/ior/*")
+        self.flags_write = self.params.get(
+            "flags_write", "/run/ior/copy_procs/*")
+        self.flags_read = self.params.get(
+            "flags_read", "/run/ior/copy_procs/*")
+
+        # Setup the directory structures
+        self.posix_test_path = join(self.workdir, "posix_test") + sep
+        self.posix_test_path2 = join(self.workdir, "posix_test2") + sep
+        self.posix_test_file = join(self.posix_test_path, self.test_file)
+        self.posix_test_file2 = join(self.posix_test_path2, self.test_file)
+        self.daos_test_file = join("/", self.test_file)
+
+        # Create the directories
+        cmd = "mkdir -p '{}' '{}'".format(
+            self.posix_test_path,
+            self.posix_test_path2)
+        self.execute_cmd(cmd)
+
+    def tearDown(self):
+        """Tear down each test case."""
+        # Remove the created directories
+        cmd = "rm -rf '{}' '{}'".format(
+            self.posix_test_path,
+            self.posix_test_path2)
+        self.execute_cmd(cmd)
+
+        # Stop the servers and agents
+        super(CopyProcsTest, self).tearDown()
+
+    def test_copy_procs(self):
+        """
+        Test Description:
+            DAOS-5659: Verify multi-process (rank) copying.
+        Use Cases:
+            Create pool.
+            Crate POSIX container1 and container2 in pool.
+            Create a single 100M file in container1 using ior.
+        :avocado: tags=all,datamover,pr
+        :avocado: tags=copy_procs
+        """
+        # Create pool and containers
+        pool1 = self.create_pool()
+        container1 = self.create_cont(pool1)
+        container2 = self.create_cont(pool1)
+
+        # Get the varying number of processes
+        procs_list = self.params.get(
+            "processes", "/run/datamover/copy_procs/*")
```

Comment:
Can be removed and `self.processes` can be used defined under DataMoverTestBase

Reply:
But this is a list of different numbers of processes, where each value in that list is a different test case.
I also chose not to use `mux` here, because starting and stopping the server takes more time than just running the test cases in a loop.